src/main/java/edu/agh/wfiis/designpatterns/patternsUsageExample/App.java

```diff
@@ -0,0 +1,39 @@
+package edu.agh.wfiis.designpatterns.patternsUsageExample;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.ApplicationContext;
+
+@SpringBootApplication
+public class App {
+    private static ApplicationContext applicationContext;
+
+    public static void main(String[] args) {
+        applicationContext = SpringApplication.run(App.class);
+        performExample();
+    }
+
+    private static void performExample() {
+        DesignPatternsManager manager = (DesignPatternsManager) applicationContext.getBean("designPatternsManager");
+
+        CreatingDesignPattern builderCreator = (DesignPattern.Builder) applicationContext.getBean("designPattern.Builder");
```

Comment:
In our opinion in this particular situation it is not neccessary to use Spring Framework because it is more complicated to analyze and not essential to understand design patterns, but it is at yours discretion.